doc/dev/credscan_process.md

```diff
@@ -76,9 +80,10 @@ section of this guide.
 
 
 [aggregate_reports]: https://dev.azure.com/azure-sdk/internal/_build?definitionId=1401&_a=summary
+[aggregate_reports_output]: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1411446&view=logs&j=9e400fad-ff47-5b38-f9dc-cae2431972da&t=8613334a-c306-55ea-63ff-80c6e8e0a0ca
```

Comment:
idk if this is like other builds where they disappear after a period of time, but if it is, then this will eventually point to a 404. Would a screenshot work instead?

Reply:
That's a good question to raise! I wondered the same thing, but Laurent showed me how to mark a build to be retained -- the pipeline runs linked to in the doc have been manually retained so we won't have to worry about that 🙂 A screenshot might be helpful though, even though there's an abstracted example. I'll make a note to consider that!

Reply:
I didn't know you could retain builds, that's really helpful! thanks for letting me know